src/components/ChangePlan/Checkout/ContactInformation/ContactInformation.js

```diff
@@ -80,6 +80,19 @@ export const ContactInformation = InjectAppServices(
       }
     };
 
+    const complete = (contactInformation) => {
```

Comment:
With this issue resolved this is ready to merge @jhoffman-ms 🎉

Reply:
Done